draft-irtf-cfrg-opaque.md

```diff
@@ -1608,6 +1608,14 @@ to support implementations which provide mitigations against client enumeration
 - Deserialization checks: When parsing messages that have crossed trust boundaries (e.g.
 a network wire), implementations should properly handle all error conditions covered in
 {{I-D.irtf-cfrg-voprf}} and abort accordingly.
+- Additional client-side entropy: OPAQUE supports the ability to incorporate the
+client identity alongside the password to be input to the OPRF. This provides additional
+client-side entropy which can supplement the entropy that should be introduced by the
+server during an honest execution of the protocol.
+- Client-side authentication of the server: Note that online guessing attacks
+(against any Asymmetric PAKE) can be done from both the client side and the server side.
```

Comment:
I think this needs some more detail to be useful. In particular, I'd try to include that a malicious server can simulate honest responses in an attempt to learn the client's password.